toolkit/imageconfigs/packagelists/base-image-packages.json

```diff
@@ -0,0 +1,27 @@
+{
+    "packages": [
```

Comment:
Wonder if this can be in `core-packges.spec` as a `core-packages-minimal` or something like that. I don't mind it either way. I do see some differences like `initramfs` which isn't included in the core-packages-* lists...

Reply:
I thought about that, but then we are still trying to create another smaller subset for recovery os scenarios, and this will not have some of the stuff in here (ie: cloud-init), that would be the true `minimum`. I like the idea of putting that in `core-packages.spec` 👍